unique_toolkit/unique_toolkit/language_model/schemas.py

```diff
@@ -82,6 +84,18 @@ def __getitem__(self, item):
         return self.root[item]
 
 
+class Messages(RootModel):
```

Comment:
there is already such a class 'LanguageModelMessages'. The difference is that it does not take ChatMessage. Should we not update the other one. Otherwise we have multiple classes for the same purpose